### ISSUE TITLE
Does it work, monster?

### ISSUE THREAD

**Issue by keerthanpg** [nuwandavek/iplLivescore#1]
Does it work, monster?

**Comment by nuwandavek** [comment]
No, I've shut down the heroku app, ya, used it only during IPL as I didn't want to keep switching tabs. Also, I didn't use the api as they asked for a lot of money. I scraped it off the espn page. :P So, not sure if the tags and named will be the same again. Should work with minimal changes though. :)
